projects/sample/initialize.py

```diff
@@ -62,16 +124,34 @@ def main(argv=None):
                            help='retrieve files from the remote server')
     args = argParser.parse_args(argv)
 
+    # load the experiment configuration file
     cfg = utils.loadConfigFile(args.config)
 
+    # open up the communication pipe using 'projectInterface'
     projectComm = projUtils.initProjectComm(args.commpipe, args.filesremote)
+    
+    # initiate the 'fileInterface' class, which will allow you to read and write 
+    #   files and many other things using functions found in 'fileClient.py'
+    #   INPUT:
+    #       [1] args.filesremote (to retrieve dicom files from the remote server)
+    #       [2] projectComm (communication pipe that is set up above)
     fileInterface = FileInterface(filesremote=args.filesremote, commPipes=projectComm)
 
-    # Do processing
+    # now that we have the necessary variables, call the function 'initialize' in
+    #   order to actually start reading dicoms and doing your analyses of interest!
+    #   INPUT:
+    #       [1] cfg (configuration file with important variables)
+    #       [2] fileInterface (this will allow you to call useful variables)
+    #       [3] projectComm (communication pipe to talk with projectInterface)
     initialize(cfg, fileInterface, projectComm)
     return 0
 
 
 if __name__ == "__main__":
+    """
+    If 'initalize.py' is called from the terminal or the equivalent, then actually go
```

Comment:
Maybe revise to "If 'initalize.py' is invoked as a program then ..."
I think saying called from terminal may be confusing because mostly it is invoked from the web interface rather than a terminal.